synchronization/src/main/java/de/cyface/synchronization/SyncPerformer.java

```diff
@@ -124,7 +124,7 @@ boolean sendData(@NonNull final Http http, @NonNull final SyncResult syncResult,
         HttpURLConnection.setFollowRedirects(false);
         HttpURLConnection connection = null;
         final String fileName = String.format(Locale.US, "%s_%d.ccyf", metaData.deviceId, metaData.measurementId);
-        final String eventsFileName = String.format(Locale.US, "%s_%d_.ccyfe", metaData.deviceId, metaData.measurementId);
```

Comment:
This typo would prevent uploading as soon as the new collector is deployed. Thus, it needs to be in all new app releases which want to contact to the S2 collector (DAT/VIC).